javalib/src/main/scala/java/lang/AbstractStringBuilder.scala

```diff
@@ -4,6 +4,8 @@ import java.io.InvalidObjectException
 import java.util.Arrays
 import scala.util.control.Breaks._
```

Comment:
If any of the `breakable`s are in the hot path that could affect your performance. Sometimes they are tricky to remove. Sometimes `ScalaOps` can help, `return` with maybe a `var` or `val` or perhaps recursion. It would be nice if they weren't in there at all in the `javalib` as that is a Scala construct. See https://alvinalexander.com/scala/scala-break-continue-examples-breakable/

Reply:
Eric, rightly so. I created issue #2918 to capture your concern.

I also reviewed the file `AbstractStringBuilder.scala`. All the usages
of the problematic `breakable` occur in methods not touched by this PR.

I propose that your concern be noted but that changing unrelated, and
now recorded, Issues not be required of this PR.  Your thoughts?

Reply:
I would treat this point in a separate PR, as explained in the issues #2918 and #2919

Reply:
Concurred, most heartily